src/components/view/Navigation/index.js

```diff
@@ -36,20 +32,10 @@ class Navigation extends React.Component {
         super(props);
         this.navigateToRoute = this.navigateToRoute.bind(this);
         this.themePalette = this.props.muiTheme.palette;
+        this.state = {open: false};
     }
-    
-    buildIconMenu (actions) {
```

Comment:
Why not destructure logout right in the arguments? 

i.e. `buildIconMenu ({ logout }) {`